recipes/ggplot/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.9.4" %}
+
+package:
+  name: ggplot
+  version: {{ version }}
+
+source:
+  fn: ggplot-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/g/ggplot/ggplot-{{ version }}.tar.gz
+  md5: 0fbaa3123cb2d4b060b200a106340421
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - brewer2mpl
+    - cycler
```

Comment:
Did some rebasing to my master branch, @jakirkham but frankly I'd prefer someone else handle it with this commit.